workflow/search-filter.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Search & Filter
+---
+
+The Search & Filter features serve as the jumping-off point in your data exploration process within Middleware. Data is made available across all contexts within Middleware, allowing you to connect all data points collected.
+
+The Filter Bar and Search Fields are two of the simplest methods for sifting through your data. They are both dynamically generated based on the applications, integrations, and infrastructure you have instrumented within Middleware. These features are inclusive (i.e. union set operations) and will display data that satisfies any of the selected filters.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaTFkdDIwa295MWlhM2liNTk1ZmM0ZHI2NnZ3aTA0c2wzdnVoOHgxbyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/awGRnLE9t3NZG1sTde/giphy.gif" width="750" height="750"/>
```

Comment:
This gif seems focused on the timeline zoom rather than filtering / search. Should we change this?